src/main/java/com/dataloom/requests/HazelcastRequestsManager.java

```diff
@@ -82,4 +111,52 @@ public void submitAll( Map<AceKey, Status> statusMap ) {
     public Stream<Status> getStatuses( Stream<AceKey> requestKeys ) {
         return requestKeys.map( Util.getSafeMapper( requests ) );
     }
+
+    private void signalPermissionRequestStatusUpdates( Map<AceKey, Status> aceKeyRequestStatusMap ) {
```

Comment:
Consider: 
- Using subList instead of creating an entirely new ArrayLIst
- Inlining transmit of the signal.
- breaking it out into a function that processes one AceKey / Status pair at a time that is used a lambda argument